src/lava/magma/core/process/process.py

```diff
@@ -221,6 +221,16 @@ def __del__(self):
         """
         self.stop()
 
+    def __enter__(self):
+        """Required for "with" block."""
```

Comment:
It could be clearer here to explicitly refer to context managers.
```python
"""Executed when Process enters a "with" block of a context manager."""
```